invoicing/ts-classes/Line.ts

```diff
@@ -0,0 +1,12 @@
+import {YLine} from "invoicing";
+import {YTmoney} from "yupana/YClass";
+
+export class Line extends YLine {
+    cGetTotal(): number {
+        const price = this.product.price as YTmoney;
+        const amount = this.amount;
+
+        const total = price.amount * amount;
```

Comment:
This gives me 
> Error:(9, 38) TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.

Reply:
is this how `YTmoney` properties are used? i don't know how we're dealing with that if that's the case...

Reply:
So ```amount``` has a type ```YTinteger``` and we don't have implicit cast to do computation with that ?

Reply:
more or less... i don't believe `YTinteger` exists at runtime and it's only a javascript `number`